src/lib/Components/Countdown/Ticker.tsx

```diff
@@ -31,7 +31,7 @@ const durationSections = (duration: Duration, labels: [string, string, string, s
 ]
 
 const LabeledTimeSection: React.SFC<TimeSectionProps> = ({ time, label, timeTextProps, labelTextProps }) => (
-  <Flex alignItems="center" justifyConent="center">
+  <Flex alignItems="center" justifyContent="center">
```

Comment:
One advantage of upgrading: this was not caught as a type error before 👏

Reply:
nice one